src/python/pants/backend/awslambda/python/targets.py

```diff
@@ -6,15 +6,28 @@
 
 
 class PythonAwsLambdaHandler(StringField):
-    """Lambda handler entrypoint (module.dotted.name:handler_func)."""
+    """AWS Lambda handler entrypoint (module.dotted.name:handler_func)."""
 
     alias = "handler"
     required = True
     value: str
 
 
+class PythonAwsLambdaRuntime(StringField):
```

Comment:
What about making this a `FloatField` and renaming the field to `runtime_python_version=3.7`? Then, `FloatField` will validate that it's a float, meaning it will reject values like `3` and `3.7.4`.

Reply:
A) I prefer to use AWS's names for its runtimes.
B) What if AWS introduce `3.7.4` as a valid runtime name?

Reply:
Okay, I recommend the changes proposed in https://github.com/pantsbuild/pants/pull/9564#discussion_r410477628 still. At a minimum, to do the first proposed change. All validation should live in the Field definition.